### ISSUE TITLE
Pipline to generate ome-zarr format

### ISSUE THREAD

**Issue by manzt** [ome/ome-zarr-py#24]
I know the format is experimental, but are there available pipelines to generate the zarr format which adheres to the ome-metadata spec? I've tried `bioformats2raw`, which seems related, but I don't see the corresponding keys in the `.zattrs` file.

Many thanks in advance! Really like this project.

**Comment by joshmoore** [comment]
Hi @manzt. To match the format outlined in https://github.com/ome/omero-ms-zarr/blob/master/spec.md, add the options:

```
--file_type=zarr --dimension-order=XYZCT
```

**Comment by manzt** [comment]
Thank you! Is the `omero` field added in the `.zattrs` as well? I've been working to get blosc working in the browser and am trying to generate some examples!

**Comment by joshmoore** [comment]
No, the `omero` field is added during post-processing. Are you looking for any examples in particular, or would downloading some via `ome_zarr download` get you what you need?

We'll definitely be showing some existing examples at https://www.openmicroscopy.org/events/ome-community-meeting-2020/ (registration closes today! :wink:)

But even if you can't join, I can provide more examples like:

```
ome_zarr info https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr
```

**Comment by manzt** [comment]
Singed up -- looking forward to it! Thanks

**Comment by dstansby** [comment]
Does the example at the top of https://ome-zarr.readthedocs.io/en/stable/python.html give enough information on generating data in the ome-zarr format? If so I think this issue could be closed.
